src/GraphQLParser.ApiTests/GraphQLParser.ApiTests.csproj

```diff
@@ -5,15 +5,21 @@
   </PropertyGroup>
```

Comment:
```suggestion
    <IsPackable>false</IsPackable>
  </PropertyGroup>
```
???

Reply:
All csproj files should have `<IsPackable>false</IsPackable>` if it's not going to output nupkg files -- I think.

Reply:
(Also needed in `GraphQLParser.Tests.csproj`)

Reply:
`IsPackable` is always false for test projects so we need to set it explicitly only for a few projects like benchmarks.